vue-suggestions.vue

```diff
@@ -53,8 +57,20 @@
     methods: {
 
       initSuggestion() {
-        const options = Object.assign({}, this.options, { onSelect: this.onSelect });
-        $(this.$el).suggestions(options);
+        this.callbacks.add(this.onSelect)
+        this.callbacks.add(this.options.onSelect || jQuery.noop)
+        const options = Object.assign({}, this.options, {
+          onSelect: suggestion => this.callbacks.fire(suggestion)
+        });
+        jQuery(this.$el).suggestions(options);
+      },
+
+      /**
```

Comment:
Delete this comment, please. I think that it is not necessary.

Reply:
Well, this is a not big deal with the comment, I hope... :smile_cat: I always have the trouble with finding the documentation about the plugin.

Reply:
You already added link to docs in readme. I think that's enough.